app/views/pages/accessibility.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Accessibility statement" %>
```

Comment:
I'm not sure if Will, will want this as:
Accessibility statement - Register a flood risk activity exemption - GOV.UK
?

Reply:
This is the same as on the WCR page.